src/System.Text.Json/tests/WritableJsonApiTests.cs

```diff
@@ -49,52 +58,73 @@ private static class EmployeesDatabase
                             {
-                                "software developers", new JsonObject()
                                 {
+                                    "full time employees", new JsonObject()
                                     {
-                                        "full time employees", new JsonObject()
-                                        {
-                                            EmployeesDatabase.GetNextEmployee(),
-                                            EmployeesDatabase.GetNextEmployee(),
-                                            EmployeesDatabase.GetNextEmployee(),
-                                        }
-                                    },
+                                        EmployeesDatabase.GetNextEmployee(),
+                                        EmployeesDatabase.GetNextEmployee(),
+                                        EmployeesDatabase.GetNextEmployee(),
+                                    }
+                                },
+                                {
+                                    "intern employees", new JsonObject()
                                     {
-                                        "intern employees", new JsonObject()
-                                        {
-                                            EmployeesDatabase.GetNextEmployee(),
-                                            EmployeesDatabase.GetNextEmployee(),
-                                        }
+                                        EmployeesDatabase.GetNextEmployee(),
+                                        EmployeesDatabase.GetNextEmployee(),
                                     }
                                 }
-                            },
+                            }
+                        },
+                        {
+                            "HR", new JsonObject()
                             {
-                                "HR", new JsonObject()
                                 {
-                                    {
-                                        "full time employees", new JsonObject(EmployeesDatabase.GetTenBestEmployees())
-                                    }
+                                    "full time employees", new JsonObject(EmployeesDatabase.GetTenBestEmployees())
                                 }
                             }
                         }
                     }
-                };
+                );
+
+                return manager;
             }
 
             public static bool CheckSSN(string ssnNumber) => true;
+            public static void PerformHeavyOperations(JsonElement employee) { }
+        }
+
+        /// <summary>
+        /// Helper class simulating sending Json files via network
+        /// </summary>
+        private static class Mailbox
+        {
+            public static void SendEmployeeData(JsonElement employeeData) { throw null; }
```

Comment:
These void methods seem like they could just be no-op.  Why are you adding test methods that throw null?

Reply:
Right, I will change it, I was in the trance of adding `throw null` to every method I am not implementing currently.